Lint raises `max-len` warnings.

The linter shows me `max-len` warnings (see below).

Should we disable that lint rule ?

```
$ yarn lint
yarn run v1.22.17
$ eslint --ext ts generators utils

generator-project/generators/root/index.ts
  64:1  warning  This line has a length of 131. Maximum allowed is 120  max-len
  67:1  warning  This line has a length of 156. Maximum allowed is 120  max-len

generator-project/generators/utils/database/index.ts
  45:1  warning  This line has a length of 127. Maximum allowed is 120  max-len

generator-project/utils/ansible/vault.ts
  60:10  warning  Prefer default export  import/prefer-default-export

generator-project/utils/circleci/Config.ts
  86:1  warning  This line has a length of 124. Maximum allowed is 120  max-len

generator-project/utils/circleci/Workflow.ts
  34:1  warning  This line has a length of 132. Maximum allowed is 120  max-len

generator-project/utils/circleci/mergeConfig.ts
   7:1  warning  This line has a length of 130. Maximum allowed is 120  max-len
  27:1  warning  This line has a length of 128. Maximum allowed is 120  max-len

generator-project/utils/validation/validateEmail.ts
  3:1  warning  This line has a length of 149. Maximum allowed is 120  max-len

generator-project/utils/validation/validateProjectName.test.ts
  4:1  warning  This line has a length of 134. Maximum allowed is 120  max-len
  8:1  warning  This line has a length of 126. Maximum allowed is 120  max-len
```

--- a/generators/root/index.ts
+++ b/generators/root/index.ts
@@ -61,10 +61,28 @@ class RootGenerator extends BaseGenerator {
         await this.spawnCommand('git', ['init']);
 
         if (!(await this.#spawnTest('git', ['remote', 'get-url', 'origin']))) {
-            await this.spawnCommand('git', ['remote', 'add', 'origin', `git@github.com:${this.config.get('repositoryName')}.git`]);
+            await this.spawnCommand(
+                'git',
+                [
+                    'remote',
+                    'add',
+                    'origin',
+                    `git@github.com:${this.config.get('repositoryName')}.git`,
+                ],
+            );
         }
 
-        await this.spawnCommand('git', ['update-index', '--add', '--cacheinfo', '160000', '4d1ffdcd4bc254bcc61fd85fc176d07b64d2d464', 'ansible/roles-lib']);
+        await this.spawnCommand(
+            'git',
+            [
+                'update-index',
+                '--add',
+                '--cacheinfo',
+                '160000',
+                '4d1ffdcd4bc254bcc61fd85fc176d07b64d2d464',
+                'ansible/roles-lib',
+            ],
+        );
     }
 
     /**

--- a/generators/utils/database/index.ts
+++ b/generators/utils/database/index.ts
@@ -42,7 +42,10 @@ class DatabaseUtilGenerator extends BaseGenerator<DatabaseUtilGeneratorOptions> 
                 ),
             );
         } else {
-            this.log('Skipping production database configuration because we couldn\'t find the expected directory structure.');
+            this.log(
+                'Skipping production database configuration because '
+                + 'we couldn\'t find the expected directory structure.',
+            );
         }
     }
 }

--- a/utils/circleci/Config.ts
+++ b/utils/circleci/Config.ts
@@ -83,7 +83,10 @@ class Config {
         raw.jobs = this.jobs;
         raw.workflows = {
             version: this.workflowsVersion,
-            ...map<Record<string, Workflow>, Record<string, any>>((workflow: Workflow) => workflow.toRaw(), this.workflows),
+            ...map<Record<string, Workflow>, Record<string, any>>(
+                (workflow: Workflow) => workflow.toRaw(),
+                this.workflows,
+            ),
         };
 
         return raw;

--- a/utils/circleci/Workflow.ts
+++ b/utils/circleci/Workflow.ts
@@ -31,7 +31,9 @@ class Workflow {
 
     toRaw(): any {
         return {
-            jobs: Object.entries(this.jobs).map(([name, config]) => (Object.keys(config).length === 0 ? name : { [name]: config })),
+            jobs: Object.entries(this.jobs).map(
+                ([name, config]) => (Object.keys(config).length === 0 ? name : { [name]: config }),
+            ),
         };
     }
 }

--- a/utils/circleci/mergeConfig.ts
+++ b/utils/circleci/mergeConfig.ts
@@ -4,7 +4,10 @@ import Workflow from './Workflow';
 /**
  * Returns keys that are present in either objects
  */
-const allKeys = (first: object, second: object): string[] => Array.from(new Set([...Object.keys(first), ...Object.keys(second)]));
+const allKeys = (first: object, second: object): string[] => Array.from(new Set([
+    ...Object.keys(first),
+    ...Object.keys(second),
+]));
 
 /**
  * Merge two configs together
@@ -24,22 +27,23 @@ const mergeConfig = (first: Config, second: Config): Config => new Config({
         ...second.jobs,
     },
     workflowsVersion: first.workflowsVersion,
-    workflows: Object.fromEntries(allKeys(first.workflows, second.workflows).map((workflowName: string): [string, Workflow] => {
-        if (!(workflowName in first.workflows)) {
-            return [workflowName, second.workflows[workflowName]];
-        }
+    workflows: Object.fromEntries(allKeys(first.workflows, second.workflows)
+        .map((workflowName: string): [string, Workflow] => {
+            if (!(workflowName in first.workflows)) {
+                return [workflowName, second.workflows[workflowName]];
+            }
 
-        if (!(workflowName in second.workflows)) {
-            return [workflowName, first.workflows[workflowName]];
-        }
+            if (!(workflowName in second.workflows)) {
+                return [workflowName, first.workflows[workflowName]];
+            }
 
-        return [workflowName, new Workflow({
-            jobs: {
-                ...first.workflows[workflowName].jobs,
-                ...second.workflows[workflowName].jobs,
-            },
-        })];
-    })),
+            return [workflowName, new Workflow({
+                jobs: {
+                    ...first.workflows[workflowName].jobs,
+                    ...second.workflows[workflowName].jobs,
+                },
+            })];
+        })),
 });
 
 export default mergeConfig;

--- a/utils/validation/validateEmail.ts
+++ b/utils/validation/validateEmail.ts
@@ -1,5 +1,6 @@
 import { Validator } from './types';
 
+// eslint-disable-next-line max-len
 const regex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
 
 const validateEmail: Validator = (value) => {

--- a/utils/validation/validateProjectName.test.ts
+++ b/utils/validation/validateProjectName.test.ts
@@ -1,11 +1,13 @@
 import validateProjectName from './validateProjectName';
 
 test('Project name with a space is not valid', () => {
-    expect(validateProjectName('test project')).toBe('Project name can only contains lowercase alphanumerical characters and dashes');
+    expect(validateProjectName('test project'))
+        .toBe('Project name can only contains lowercase alphanumerical characters and dashes');
 });
 
 test('Project name with uppercase characters is not valid', () => {
-    expect(validateProjectName('TEST')).toBe('Project name can only contains lowercase alphanumerical characters and dashes');
+    expect(validateProjectName('TEST'))
+        .toBe('Project name can only contains lowercase alphanumerical characters and dashes');
 });
 
 test('Project name starting with a number is not valid', () => {

